card/lib/card/query/attributes.rb

```diff
@@ -170,31 +171,25 @@ def extension_type val
 
       def join_references key, val
         r = Reference.new( key, val, self )
-        joins << Join.new(:from=>self, :to=>r, :to_field=>r.infield)
-        s = nil
+        refjoin = Join.new(:from=>self, :to=>r, :to_field=>r.infield)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.